### ISSUE TITLE
Pycharm issue

### ISSUE THREAD

**Issue by camilocbarrera** [withfig/fig#2043]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.1 (22A400)

### Expected behaviour

The help bar should appears inside the terminal

### Actual behaviour

The autocomplete bar appears in a different place 
![image](https://user-images.githubusercontent.com/85809276/201552997-ad82cd2c-0cf8-4b87-a407-dc751fc737f8.png)

### Environment

```yaml
fig-details:
  - 2.7.8
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 10
  - mem: 16.00 GB
os-info:
  - macOS 13.0.1 (22A400)
environment:
  - shell: /bin/zsh
  - terminal: pycharm
  - cwd: /Users/cris/data_team/repos/DL-Dbt
  - exe-path: /Users/cris/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 1f20d552-940e-4c2f-828b-ea2ca66a5a3c
    - FIG_HOSTNAME: just-another-macbook.local
    - FIG_TERM: 2.7.8
    - PATH: /opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/Users/cris/.fig/bin:/Users/cris/.local/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 33135
    - FIG_PARENT: 1f20d552-940e-4c2f-828b-ea2ca66a5a3c
    - FIG_SET_PARENT: 1f20d552-940e-4c2f-828b-ea2ca66a5a3c
```

**Comment by mschrage** [comment]
Can you run `fig doctor` in the PyCharm window?

Also is this instance of PyCharm installed via Toolbox? If so, this might be related to #2018

**Comment by grant0417** [comment]
This should be resolved now in Fig 2.10, let me know if you continue to have any issues!
